float64[] SO3
---
geometry_msgs/Pose pose
